### ISSUE TITLE
Backspace is disabled when in INSERT mode [VS:Mac]

### ISSUE THREAD

**Issue by lordfeck** [VsVim/VsVim#3020]
**Describe the bug**
Backspace is not working in INSERT mode.

**To Reproduce**
Using VS: Mac, go to insert mode. Press backspace. The system gives a bell and does not delete the preceding character.

**Expected behavior**
Backspace deletes the preceding char.

**Environment (please complete the following information):**

- Visual Studio version: 17.3.6 (Mac)
- VsVim version: VsVim 2.8.0.19
- Programming Language: C#

**Comment by nosami** [comment]
Check out my [example `~/vsvimrc` file](https://github.com/VsVim/VsVim/wiki/MacOS-guide#example-vim-configuration)... Especially the `set backspace` part

**Comment by lordfeck** [comment]
@nosami That fixed it, thanks.

**Comment by leftbones** [comment]
Since this isn't closed yet, I want to ask... Why is this required only on the Mac version?

I've been using VsVim on Windows for awhile now and never had to mess with a `.vsvimrc` at all, then I decide to try it out on Mac and backspace isn't working. Super annoying, the defaults should be the same on both platforms.

**Comment by nosami** [comment]
> Since this isn't closed yet, I want to ask... Why is this required only on the Mac version?
> 
> I've been using VsVim on Windows for awhile now and never had to mess with a `.vsvimrc` at all, then I decide to try it out on Mac and backspace isn't working. Super annoying, the defaults should be the same on both platforms.

Yes, I agree. It is just a question of getting around to it when there are a million other things that I would rather be doing :)
